extensions/amp-bind/0.1/amp-state.js

```diff
@@ -48,8 +49,11 @@ export class AmpState extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    user().assert(isExperimentOn(this.win, 'amp-bind'),
-        `Experiment "amp-bind" is disabled.`);
+    // Don't check experiment when testing.
```

Comment:
Why don't we check for the experiment flag when running tests? I don't think that's inscrutable.